test/unit/server.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+var chai = require('chai');
+chai.use(require('chai-as-promised'));
+var assert = chai.assert;
+var sinon = require('sinon');
+
+var Promise = require('bluebird');
+var Worker = require('../../lib/worker');
+var hermes = require('runnable-hermes');
+var noop = require('101/noop');
+var ponos = require('../../');
+
+var tasks = {
+  'test-queue-01': worker,
+  'test-queue-02': worker
+};
+function resolvePromise () { return Promise.resolve(); }
+function worker (job, done) {}; // eslint-disable-line no-unused-vars
+
+describe('Ponos Server', function () {
+  var server;
+  before(function () { sinon.stub(Worker, 'create'); });
+  beforeEach(function () {
+    server = ponos.server({ queues: Object.keys(tasks) });
+    sinon.stub(server.hermes, 'connectAsync', resolvePromise);
+    sinon.stub(server.hermes, 'subscribeAsync', resolvePromise);
+  });
+  afterEach(function () {
+    server.hermes.connectAsync.restore();
+    server.hermes.subscribeAsync.restore();
+  });
+  after(function () { Worker.create.restore(); });
+
+  describe('Constructor', function () {
```

Comment:
How are you possibly getting full coverage here? You're not testing any of the `RABBITMQ_*` environment variables, nor determining if they actually default correctly. Is it possible that istanbul is not smart enough to catch in-line logical ORs?